apps/ledger-live-mobile/e2e/models/manager/managerPage.ts

```diff
@@ -8,4 +9,8 @@ export default class ManagerPage {
   async openViaDeeplink() {
     await openDeeplink(baseLink);
   }
+
+  async waitForManagerPageToLoad() {
```

Comment:
Remove async and return the promise

Reply:
it's a mistake. i'll add the missing await in the code 👍